src/index.css

```diff
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
+
+h1{
+  text-align: center;
+}
+
+.input-BarreRecherche{
+  width: 90%;
+  height: 50px;
+  text-align: center;
+  font-size: 120%;
+  margin: 0 5%;
+
+}
+
+.lesImages{
+display: flex;
+flex-wrap: wrap;
+justify-content: center;
+width: 98%;
+margin: 0 auto;
```

Comment:
@malvena15 Since we are using flex, could we remove the `width` and `margin` properties?

Reply:
Absolutely, I only use the width and the margin to keep a white space around my container. It's more a personal choice, I think